### PR TITLE
Clarification comments

### PR DESCRIPTION
**Pull request by filiptrplan**
* added comments to top of client-side files for easier file navigation and to explain the purpose of each file.

* moved all the commented-out code to `comments.txt` for more clarity

### CODE REVIEW

File: world-client/src/features/feature.js [hunk 1]
```diff
@@ -1,3 +1,38 @@
+/**
```
Comment (jakakonda) [inline world-client/src/features/feature.js]: Kaj če bi naredili eno mapo "docs" in notri metali takšne datoteke v Markdown? @davidmohar , thoughts? Enako zgornje.

Comment (davidmohar) [reply world-client/src/features/feature.js]: Agreed

Comment (filiptrplan) [reply world-client/src/features/feature.js]: Se mi zdi, da bi tako manj ljudi sploh prebralo te komentarje. Že tako komaj berejo dokumentacijo. Komentarji na vrhu datotek so mogoče malo grdi ampak tako zagotovimo, da jih bodo vsi (upam) vsaj enkrat prebrali.

UPDATE: Bom poskusil markdown rešitev pa vidimo kako gre.

**Comment by filiptrplan** [comment]
@davidmohar @jakakonda 
Created a `docs` folder for documentation,
requesting a re-review.

**Comment by filiptrplan** [comment]
Seems this won't be merged.

**Pull request closed without merging**